src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -103,7 +104,11 @@ else()
     # it needs to build with coreclr in order to be linkable with it on Windows
     # coreclr and static libraries must match because linker does not permit mixing modules
     # with different C runtimes (i.e. Debug vs. Release)
-    install (TARGETS System.Globalization.Native-Static DESTINATION lib)
+    if(CLR_CMAKE_TARGET_UNIX_WASM)
```

Comment:
Other libraries shims use `DESTINATION ${STATIC_LIB_DESTINATION}` for the static libraries. Would it fix the problem?

Reply:
I am trying to clean this up in https://github.com/dotnet/runtime/pull/47127 . We can merge this change in the meantime.

Reply:
Thanks!